index.md

```diff
@@ -24,6 +27,6 @@ existing and hypothetical materials, many of which have appeared online:
 - [the Joint Automated Repository for Various Integrated Simulations](https://jarvis.nist.gov)
 - ...
 
-The **Open Databases Integration for Materials Design** (OPTIMADE) consortium
-aims to make materials databases interoperational by developing a common REST API.
+Should you wish to cite the OPTIMADE specification, please use
 
+> Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195051](http://doi.org/10.5281/zenodo.4195051)
```

Comment:
Pending discussion in the spec PR, we will decide whether to link to the unversioned DOI (the suggestion below) or keep it at v1.

```suggestion
> Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195050](http://doi.org/10.5281/zenodo.4195050)
```

Reply:
Maybe HTTPS instead of HTTP here too?